schedule/yast/modify_existing_partition_opensuse.yaml

```diff
@@ -37,4 +37,10 @@ test_data:
     # part_size is the size we input in partitioner, lsblk_expected_size_output is what we'll use for validation.
     part_size: '10GiB'
     lsblk_expected_size_output: '10G'
-
+  swap:
```

Comment:
Except for `existing_partition` you could have a `test_data file` with the rest of the keys (unifying both to 11GB) and only have in the schedule that key and the `!include`. WDYT?

Reply:
Makes sense

Reply:
Done